src/server.py

```diff
@@ -416,7 +414,8 @@ def run_seta_details_query():
     date = request.args.get("date", "")
     active = request.args.get("active", 0)
     buildbot = request.args.get("buildbot", 0)
-    branch = request.args.get("branch", '')
```

Comment:
alright, that's really a issue need to solve. I will figure it out within these two day and add it into this patch.